operator/src/main/java/oracle/kubernetes/operator/rest/model/AdmissionRequest.java

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2022, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator.rest.model;
+
+import java.util.Map;
+
+import com.google.gson.annotations.Expose;
+
+/**
+ * AdmissionRequest represents a Kubernetes admission request inside an AdmissionReview object sent by the
+ * Kubernetes ApiServer upon invoking an admission webhook.
+ */
```

Comment:
Technically, you did indeed add javadoc which describes the class; I suppose I should have been more specific. I am looking for a description of the class that does not assume that the reader would have known what it is by the name. This javadoc doesn't actually provide useful information.

The terms 'admission request' and 'admission review' are Kubernetes concepts that not every operator developer is going to know automatically. I'm not looking for a long dissertation here - just something that would give somebody tasked to maintain this code a place to start understanding it. Ideally, you could provide a link to the K8S docs, if they were any good. Technical documentation is a real problem on many open source projects.

I'm thinking something more like the following - which might not be correct. I assume you understand what is going on here better than I do:

The process of adding a resource to Kubernetes is called "admission" and there are mechanisms both to validate and to modify resources when they seek admission. This object is created by Kubernetes and sent by it as a request to an operator-installed webhook, which can then decide whether the creation should be allowed, and what modifications, if any, need to be made.

Of course, you would also need javadoc for the fields to explain how they support what this object does. The goal, remember, is to make it as easy as possible for the next developer to pick up the work.

Please understand this a as a general comment on why Javadoc is important, and how it should be written,